packages/mui-material/src/styles/experimental_extendTheme.js

```diff
@@ -1,7 +1,6 @@
 import { deepmerge } from '@mui/utils';
 import { colorChannel } from '@mui/system';
 import createThemeWithoutVars from './createTheme';
-import createPalette from './createPalette';
 
 export const defaultOpacity = {
```

Comment:
@siriwatknp I think we need to have different default opacity values for light and dark theme to simplify the implementaiton. Now we have different values for `inputTouchBottomLine` and `placeholder`, but in MD even the hover opacity should have different value in [light](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createPalette.js#L37) vs [dark](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createPalette.js#L67) mode.

Reply:
This difference is the reason why in the `Button.js` I am using the opacity from the `action.hoverOpacity`.

Reply:
@mnajdova I think we should not add the `active`, `disabled` ... to the `theme.vars.opacity` at this point because we already have `theme.vars.palette.xxxOpacity` which are already configurable on both light and dark color schemes. 

Adding theme to `theme.vars.opacity` could create confusion and we are not using them in the migration at all. What do you think?

Reply:
Sure, let's skip them for now. Long term I think we should have all opacities under one object, that was my intention with the comment.